library/common/config/config.cc

```diff
@@ -0,0 +1,387 @@
+// NOLINT(namespace-envoy)
+#include "library/common/config/internal.h"
+#include "library/common/config/templates.h"
+
+const char* platform_filter_template = R"(
+        - name: envoy.filters.http.platform_bridge
+          typed_config:
+            "@type": type.googleapis.com/envoymobile.extensions.filters.http.platform_bridge.PlatformBridge
+            platform_filter_name: {{ platform_filter_name }}
+)";
+
+const char* native_filter_template = R"(
+        - name: {{ native_filter_name }}
+          typed_config: {{ native_filter_typed_config }}
+)";
+
+const char* route_cache_reset_filter_insert = R"(
+        - name: envoy.filters.http.route_cache_reset
+          typed_config:
+            "@type": type.googleapis.com/envoymobile.extensions.filters.http.route_cache_reset.RouteCacheReset
+)";
+
+const int custom_cluster_indent = 2;
```

Comment:
I think I am missing how this is used

Reply:
It's not used yet, but will be used in p3 to help avoid brittle assumptions about indentation in remote code locations (see my TODO elsewhere in this PR).